wrappers/Python/setup.py

```diff
@@ -448,6 +484,7 @@ def find_cpp_sources(root=os.path.join('..', '..', 'src'), extensions=['.cpp'],
                 "Operating System :: OS Independent",
                 "Topic :: Software Development :: Libraries :: Python Modules"
                 ],
+               setup_requires=['Cython'],
```

Comment:
this was a failed attempt at getting cibuildwheel to install it. I guess I could remove it.

Reply:
Yeah, I would.  It is needed to build, and the ``setup_requires`` is a defective setuptools feature